src/main/java/frc/robot/Robot.java

```diff
@@ -24,39 +35,84 @@
   private String m_autoSelected;
   private final SendableChooser<String> m_chooser = new SendableChooser<>();
 
+  // Talons and Victors
   /**
-   * This function is run when the robot is first started up and should be
-   * used for any initialization code.
+   * @TODO add how the ports works
+   */
+  WPI_TalonSRX rightTalon = new WPI_TalonSRX(6);
+  WPI_TalonSRX leftTalon = new WPI_TalonSRX(5);
+  WPI_VictorSPX rightVictor = new WPI_VictorSPX(1);
+  WPI_VictorSPX leftVictor = new WPI_VictorSPX(4);
+
+  // Differential Drive
+  DifferentialDrive differentialDrive = new DifferentialDrive(leftTalon, rightTalon);
+
+  // Joysticks
+  Joystick driveController = new Joystick(0);
+  Joystick manipulateController = new Joystick(1);
+
+  // Drive speed toggles
```

Comment:
```suggestion
  // Drive Motors
```
Will be analogous to `Shooting Motors` later on.

Reply:
Implemented.